### PR TITLE
Fatima: Company Worker System

### PR DESCRIPTION
**Pull request by FatimaFarah93**
Created abstract class workerFatima and subclasses for office worker, factory worker and remote worker and implemented abstract methods. Created a main class to execute the code.

### CODE REVIEW

File: src/main/java/session9inheritanceabstract/taskentries/FactoryWorkerFatima.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package session9inheritanceabstract.taskentries;
+
+public class FactoryWorkerFatima extends WorkerFatima{
+
+    public FactoryWorkerFatima(String name, String department) {
+        super(name, department);
+    }
+
+    void work() {
```
Comment (owynowyn) [inline src/main/java/session9inheritanceabstract/taskentries/FactoryWorkerFatima.java:9]: Could maybe add the @Override tag here to clarify this is an overridden method

File: src/main/java/session9inheritanceabstract/taskentries/FactoryWorkerFatima.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        System.out.println(name + " is currently using the machinery");
+    }
+
+    void takeBreak() {
```
Comment (owynowyn) [inline src/main/java/session9inheritanceabstract/taskentries/FactoryWorkerFatima.java:13]: Could maybe add the @Override tag here to clarify this is an overridden method

Comment (owynowyn) [reply src/main/java/session9inheritanceabstract/taskentries/FactoryWorkerFatima.java:13]: Same on the other classes methods actually

**Review by owynowyn — CHANGES_REQUESTED** [review]
Great work Fatima, not an essential change to make but adding the @Override tag to your methods will really help someone else see that these methods are overridden from a parent class. Again, not major but hopefully just a quick change to make 👍 🟢

**Comment by FatimaFarah93** [comment]
Hey @owynowyn just added override to the worker java classes in the latest commit.

**Review by owynowyn — APPROVED** [review]
Great work Fatima!

**Pull request merged**